Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -172,7 +172,7 @@ public async Task Execute_PullRequest_SonarCloud()
         [TestMethod]
         public async Task Execute_PullRequest_SonarQubePrior99()
         {
-            using var sut = new CacheProcessor(Mock.Of<ISonarWebService>(x => x.GetServerVersion() == Task.FromResult(SonarQubeVersion98)), CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=master"), Mock.Of<IBuildSettings>(), logger);
```

Comment:
For `ServerVersion` we did the same. It's just a property now since the only thing it does is to return a backing field value.